### ISSUE TITLE
Cannot get script to run...

### ISSUE THREAD

**Issue by pastorjustin** [jacksegal/google-compute-snapshot#36]
I cannot get this script to run.  It has an issue with the --filter argument.  Anyone else having this issue?

2018-03-01 19:45:21: Start of Script
2018-03-01 19:45:21: Start of createSnapshotWrapper
usage: gcloud compute disks list  [NAME ...] [optional flags]
ERROR: (gcloud.compute.disks.list) unrecognized arguments: --filter (did you mean '--limit'?)
usage: gcloud compute disks snapshot  DISK_NAME [DISK_NAME ...] [optional flags]
ERROR: (gcloud.compute.disks.snapshot) argument --zone: expected one argument
2018-03-01 19:45:24: Start of deleteSnapshotsWrapper
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ... 
            [-f fmt date | [[[[[cc]yy]mm]dd]HH]MM[.ss]] [+format]
usage: gcloud compute snapshots list  [NAME ...] [optional flags]
ERROR: (gcloud.compute.snapshots.list) unrecognized arguments: --filter=name~'gcs-.*5205702950000303181-.*'
2018-03-01 19:45:24: End of Script

**Comment by jacksegal** [comment]
Hi @pastorjustin, 

It's still working for me - are you using the latest version?  You can re-run the install steps if you're not sure:

```
cd ~
wget https://raw.githubusercontent.com/jacksegal/google-compute-snapshot/master/gcloud-snapshot.sh
chmod +x gcloud-snapshot.sh
sudo mkdir -p /opt/google-compute-snapshot
sudo mv gcloud-snapshot.sh /opt/google-compute-snapshot/
sudo /opt/google-compute-snapshot/gcloud-snapshot.sh
```

If that still doesn't work, can you run the following commands and let me know the output?

- getInstanceName: 
```
curl -s "http://metadata.google.internal/computeMetadata/v1/instance/hostname" -H "Metadata-Flavor: Google"
```
- getInstanceId: 
```
curl -s "http://metadata.google.internal/computeMetadata/v1/instance/id" -H "Metadata-Flavor: Google"
```
- getInstanceZone: 
```
curl -s "http://metadata.google.internal/computeMetadata/v1/instance/zone" -H "Metadata-Flavor: Google"
```

Thanks,
Jack

**Comment by pastorjustin** [comment]
I got this working… I found out all my issues were with the linux machine I was running.  I fired up a brand new one and everything worked perfectly.

Thank you,

-j

Justin Mitchell
Arienne Associates
[Description: cid:image001.png@01CBB324.6E8EFBD0]
5830 Venture Park Dr. - Suite A  |  Kalamazoo, MI  49009  |  USA
Direct: 269.359.7285
Phone:  269-329-4011 x111
jmitchell@arienne.com<mailto:jmitchell@arienne.com>
www.arienne.com<http://www.arienne.com/>
